doc/GETSTARTED.md

```diff
@@ -0,0 +1,66 @@
+# Get Started
+
+- Skill requirements
+ - Working with  Linux
```

Comment:
MacOS and BSDs are also fully supported. Windows might be supported but it is not tested much.